README.md

```diff
@@ -0,0 +1,94 @@
+Ecosystem Edge NVIDIA-CI - Golang Automation CI
+=======
+# NVIDIA-CI
+
+## Overview
+The [nvidia-ci](https://github.com/rh-ecosystem-edge/nvidia-ci) is an automation/CI framework to test NVIDIA operators, starting with the GPU Operator.
+This project is based on golang + [ginkgo](https://onsi.github.io/ginkgo) framework.
+
+### Project requirements
+* golang v1.20.x
+* ginkgo v2.x
+
+## nvidia-ci
+The [nvidia-ci](https://github.com/rh-ecosystem-edge/nvidia-ci) project is designed to test a pre-installed OpenShift Container Platform (OCP) cluster which meets the following requirements:
+
+### Mandatory setup requirements:
+* OCP cluster installed with version >=4.12
+
+### Supported setups:
+* Regular cluster 3 master nodes (VMs or BMs) 2 workers (VMs or BMs)
+* Single Node Cluster (VM or BM)
+* Public Clouds (AWS, GCP and Azure)
+
+### General environment variables
+#### Mandatory:
+* `KUBECONFIG` - Path to kubeconfig file.
+#### Optional:
+* Logging with glog
+
+We use glog library for logging. In order to enable verbose logging the following needs to be done:
+
+1. Make sure to import inittool package in your go script, per this example:
+
+<sup>
+    import (
+      . "github.com/rh-ecosystem-edge/nvidia-ci/tests/internal/inittools"
+    )
+</sup>
+
+2. Need to export the following SHELL variable:
+> export VERBOSE_LEVEL=100
+
+##### Notes:
+
+  1. The value for the variable has to be >= 100.
+  2. The variable can simply be exported in the shell where you run your automation.
+  3. The go file you work on has to be in a directory under github.com/rh-ecosystem-edge/nvidia-ci/tests/ directory for being able to import inittools.
```

Comment:
Are you sure about it? The instructions [here](https://github.com/rh-ecosystem-edge/nvidia-ci/pull/11/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R34-R38) are pulling the library directly from github (and not locally).

Reply:
We are importing the inittools package from Github